CONTRIBUTING.md

```diff
@@ -108,9 +108,15 @@ Once its done, you can click `Create Pull Request` and wait for approval.
 
 ![pull-requests-approved](https://user-images.githubusercontent.com/68053619/193328530-3cde0298-0254-4587-947a-43d51d02878f.png)
 
+#### Some useful PR commands
+
+Once a PR is created you can:
+- comment `/request_review` to a request review from the crysostatio/reviewers team.This will add a label `review-requested` which will be
```

Comment:
> crysostatio/reviewers

`cryostatio/reviewers`. Maybe including the backticks for formatting, or bolding this term, could be nice to highlight that this is actually a specific term being used.

> team.This will

`team. This will`